openshift/templates/base-images/dotnet-31.yaml

```diff
@@ -0,0 +1,61 @@
+kind: Template
```

Comment:
I'm curious why this one is needed?  Is it a template to pull in the image with your redhat account?

Reply:
Okay, so the reason I included these is because you "manually" imported them but we have no configuration files reflecting that. The whole idea of having these files in source control is to create everything needed from scratch (if needed). As it is, someone would not be able to use our configs unless the had those images imported into their environments.

Also, we will need to document how to create the pull secret needed to authenticate against RedHat's container registry. That will come in a subsequent PR

Reply:
Ah yeah perfect!  I forgot about that.  So you're just missing the information for the pull secret.